pkg/corerp/frontend/controller/secretstores/kubernetes.go

```diff
@@ -194,6 +198,14 @@ func UpsertSecret(ctx context.Context, newResource, old *datamodel.SecretStore,
 			return nil, err
 		}
 	}
+	err = options.KubeClient.Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}})
```

Comment:
Why do we need to search for the namespace again if the previous step is getting it from the app/env?

Reply:
that is correct, since its not app namespace anymore, we need to chekc if its already present and create a new one if it doesnt exist